ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -95,6 +115,88 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-heterogeneous
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
+- as: aws-ipi-sno-lvms-f28
+  cron: 25 8 18 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.2xlarge
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.15
+      LVM_OPERATOR_SUB_SOURCE: qe-app-registry
+      OCP_ARCH: arm64
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~StagerunOnly;~LSO&;STORAGE&
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-aws-ipi-sno-lvms
+- as: azure-ipi-arm-mixarch-disc-fullypriv-f28-day2-64k-pagesize
```

Comment:
@rioliu-rh pls help confirm if this job need to be covered in upgrade ci.

Reply:
LGTM cc: @aleskandro @lwan-wanglin

Reply:
LGTM